homeassistant/components/workday/strings.json

```diff
@@ -14,19 +14,19 @@
       "options": {
         "description": "Set additional options for {name} configured for country {country}",
         "data": {
-          "excludes": "Excludes",
+          "excludes": "Days to exclude",
           "days_offset": "Offset",
-          "workdays": "Workdays",
+          "workdays": "Days to include",
           "add_holidays": "Add holidays",
           "remove_holidays": "Remove Holidays",
           "province": "Subdivision of country",
           "language": "Language for named holidays",
           "category": "Additional category as holiday"
         },
         "data_description": {
-          "excludes": "List of workdays to exclude, notice the keyword `holiday` and read the documentation on how to use it correctly",
+          "excludes": "Select which weekdays to exclude as workdays. The key `holidays` adds all days according to the settings below.",
```

Comment:
The phrase "The key `holidays` adds all days according to the settings below." would make me believe only the settings below has an impact which is not the case (as it loads all holidays from the library and below settings are actually just tuning the result of that).
I think we should just revert to the prior text --> read the docs.

Reply:
Agreed, it loads the settings from the Library for the country selected for the configuration, shown at the very top of the dialog with the respective country code.

But the remaining settings for the Library as
- Subdivision of country
- Additional category as holiday
are all found below in the dialog.

As I understand your comment, that "below" irritates you because you read this as "right below" which is the "Offset", "Add holidays" and "Remove holidays" options which come next, before the subdivision and category options.

So how about:

    Select which weekdays to exclude as workdays. 
    The key `holidays` adds those for the configured country, customizable by all the settings below.
    Read the documentation on how to use them correctly.

This fully explains the way the `holidays` key is calculated and we keep the hint to consult the docs.

Reply:
Looks like a good proposal 👍

Reply:
Great. Updated as proposed.
Thank you very much!